regexes/bots.yml

```diff
@@ -91,9 +91,17 @@
     url: 'https://www.alexa.com'
 
 - regex: 'Amazonbot'
-  name: 'Amazon Bot'
-  category: 'Crawler'
-  url: 'https://developer.amazon.com/support/amazonbot'
+  name: 'Amazonbot'
+  category: 'AI Crawler'
+  url: 'https://developer.amazon.com/amazonbot'
+  producer:
+    name: 'Amazon.com, Inc.'
+    url: 'https://www.amazon.com/'
```

Comment:
According to https://developer.amazon.com/amazonbot, there is also `Amzn-SearchBot`, which we currently don't cover. Might be good to add that as well.